libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -445,13 +444,10 @@ def stream(
         *,
         stop: Optional[list[str]] = None,
         **kwargs: Any,
-    ) -> Iterator[BaseMessageChunk]:
+    ) -> Iterator[BaseMessage]:
```

Comment:
If the model doesn't implement streaming, a BaseMessage is returned, not a BaseMessageChunk.